package.json

```diff
@@ -36,7 +36,7 @@
     "passport-http-bearer": "^1.0.1",
     "passport-local": "^1.0.0",
     "pk6parse": "^0.10.1",
-    "rc": "1.0.1",
+    "rc": "1.1.6",
```

Comment:
This was the only thing from the original update that hadn't been updated again, so I thought I may as well update it.